src/ci/docker/host-x86_64/dist-various-2/Dockerfile

```diff
@@ -32,10 +32,14 @@ RUN add-apt-repository -y 'deb https://apt.dilos.org/dilos dilos2 main'
 ENV \
     AR_x86_64_fuchsia=x86_64-fuchsia-ar \
     CC_x86_64_fuchsia=x86_64-fuchsia-clang \
+    CFLAGS_x86_64_fuchsia="--sysroot=/usr/local/core-linux-amd64-fuchsia-sdk/arch/x64/sysroot" \
```

Comment:
I think this is missing the `--target` flag as [used here](https://fuchsia.googlesource.com/infra/recipes/+/d36a3b608399d38367aead9ae9ede46f65101e61/recipes/contrib/rust_toolchain.resources/generate_config.py#144) (at least it can't hurt).

Also needs the `LDFLAGS` vars set by that script.

Reply:
Good call -- added